modules/list.ts

```diff
@@ -35,7 +35,11 @@ if (currentListId) {
 
 window.toggleItemPurchaseStatus =
   itemManager.toggleItemPurchaseStatus.bind(itemManager);
+window.toggleAllItemsPurchaseStatus =
```

Comment:
Defining a lot of functions on the window... this is probably the point I'd say let's look into using jquery... but once the edit actions are complete I will be building this same project with React Native.

This has been a good reminder of how nice it is to encapsulate functions inside components in frameworks like React 😅 - which is something we can still do by using the document instead of the window, I think. Like:

```ts
document.getElementById('id').addEventListener('click', () => {
functionToUse();
});
```

Reply:
Coming back to this I'd also like to note that if I did this again I'd opt for the doing something more like the above example, it's gotten a bit unwieldy managing what functions I'm throwing in the global namespace and where. Uisng the `document` syntax in the appropriate js module would be easier to find/make sense of.